### ISSUE TITLE
Player injury info

### ISSUE THREAD

**Issue by alanjeffares** [vaastav/Fantasy-Premier-League#41]
It would be useful to have some indication of if a player was injured in a given gameweek (beyond just seeing their minutes are 0). This doesn't seem to be under https://fantasy.premierleague.com/api/element-summary/123/  but it is tracked by fpl to some degree. For example chance_of_playing_next_round, chance_of_playing_this_round and news in https://fantasy.premierleague.com/api/bootstrap-static/ . However this seems to only be current info. Might this be something worth capturing? If so are historic info for these fields stored anywhere?

**Comment by vaastav** [comment]
Sadly, I have no way of going back and adding this data for past gws.
But, this can be done for the 20-21 season onwards.

**Comment by fpl-analytics** [comment]
Just wondering if this ever got added? I can't seem to find anywhere.

**Comment by Amed1710** [comment]
This is why i think it would interesting to save all JSON without any transformation so that we can go back to the historic json to extract fields.

**Comment by Amed1710** [comment]
Is that possible ?

**Comment by vaastav** [comment]
Most of the information from the JSON is usually parsed. The only thing possibly missing is the injury info and I haven't gotten around to add it.

**Comment by ScratZa** [comment]
any idea where this data can be found if not In the datasets? I'm trying to find historical data around injuries but it's evading me. :(

**Comment by vaastav** [comment]
You can try checking out the repository at every single commit in its history and then use the version of the raw json file to extract the injury data.

**Comment by zentico** [comment]
@alanjeffares, did you manage to compile the historical injury and suspension data?

**Comment by alanjeffares** [comment]
> @alanjeffares, did you manage to compile the historical injury and suspension data?

No, this is not something I will have time to do, unfortunately.
